src/sasquatchbackpack/schemas/usgs.avsc

```diff
@@ -0,0 +1 @@
+{"namespace": "lsst.example", "type": "record", "name": "$topic_name", "description": "Collection of earthquakes near the summit", "fields": [{"name": "timestamp", "type": "long"}, {"name": "id", "type": "str", "description": "unique earthquake id"}, {"name": "latitude",  "type": "float", "units": "Degrees"}, {"name": "longitude", "type": "float", "units": "Degrees"}, {"name": "depth", "type": "float", "units": "Km"}, {"name": "magnitude", "type": "float", "units": "Richter Magnitudes"}]}
```

Comment:
See the comment above about adding the namespace to the `DispatcherConfig`. So it needs to be templated here.
Please add a new line to the end of the file; you can configure vscode to do that for you.